chart/templates/dropdb.yaml

```diff
@@ -37,6 +40,10 @@ spec:
           value: {{ .Values.application.database_user }}
         - name: PGPASSWORD
           value: {{ .Values.application.database_password }}
-        command: ["dropdb", "{{ .Values.application.database_name }}"]
+        command: ["/bin/sh"]
+        args: ["-c", "until dropdb {{ .Values.application.database_name }}; do sleep 3; done; rm -rf {{ .Values.data.datadir }}/filestore/{{ .Values.application.database_name }}"]
       restartPolicy: Never
+      volumes:
+      - name: "data-odoo"
+{{ toYaml .Values.data.volumes | indent 8 }}
```

Comment:
This should rather be `Values.odoodata.volume` (because this value defines only one data value for the odoo data dir and nothing else).

One could imagine having another value for additional volumes, but then we need another value to say where they must be mounted in the container... so that will be for later.